x-pack/plugins/triggers_actions_ui/public/application/lib/validate_params_for_warnings.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { intersection, get } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import { ActionVariable } from '@kbn/alerting-plugin/common';
+
+const publicUrlWarning = i18n.translate(
+  'xpack.triggersActionsUI.sections.actionTypeForm.warning.publicUrl',
+  {
+    defaultMessage: 'Kibana missing publicUrl environment variable. Action will use relative URLs.',
```

Comment:
Couple of questions:
- Is there a reason for calling it an environment variable here? In the docs we call them settings: https://www.elastic.co/guide/en/kibana/8.6/settings.html#server-publicBaseUrl
- This seems to be a setting that isn't supported on Cloud. I hope that means it wouldn't appear in those circumstances; otherwise, we'd need to clarify that in the message too

I suggest using the full setting name. For example:

```suggestion
        comments: 'server.publicBaseUrl is not set. Actions will use relative URLs.',
```

Reply:
Thank you for looking at this!

I got this text from a mock up, so I don't think there is a specific reason for calling it environment variable.

I think this message would still show up in cloud as it's written right now.

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/149832/commits/0ede4ac5d89df52e6cdc3faff036f4806579e28c